edkrepo_manifest_parser/edk_manifest.py

```diff
@@ -648,6 +688,43 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self.__eq__(other)
 
+class _PatchSet():
+    def __init__(self, element):
+        try:
+            self.remote = element.attrib['remote']
+            self.name = element.attrib['name']
+            self.parentSha = element.attrib['parentSha']
+            self.fetchBranch = element.attrib['fetchBranch']

```

Comment:
To follow the formatting precedent set elsewhere in this file please use self.parent_sha etc... for the members of this class.